Khan/wonder-blocks
# WB-1014 - Add `style` prop to `Spring` and `Strut`

## Summary:
Integrated the style prop to the Spring and Strut components under wonder-blocks-layout. Included new examples to StyleGuidist and React Storybook demonstrating the new change.

Also changed the components call in the Styleguide config for the Layout and Clickable components to prevent Storybook components from being picked up by StyleGuidist.

Issue: https://khanacademy.atlassian.net/browse/WB-1014

## Test plan:
1. Open StyleGuidist in your browser
2. Go to http://localhost:6060/#spring (strut is right below it)
3. See the code examples to see how you can now pass styles directly to the Spring and Strut components.

You can also view the React Storybook preview link to see the new additions for Spring and Strut.

![Screen Shot 2021-05-11 at 5 10 04 PM](https://user-images.githubusercontent.com/60367213/117891031-d3db0580-b27b-11eb-8c70-46d767e95e99.png)
###### Simple SS example for Spring on StyleGuidist
